src/content/docs/en/reference/components-reference.mdx

```diff
@@ -0,0 +1,263 @@
+  height="900"
+  decoding="async"
+  loading="lazy"
+  alt="A description of my image."
+/>
+```
+#### Properties
+
+- src (required)
+- alt (required)
+- width and height (required for `public/` and remote images)
+- format
+- quality
+- densities
+- widths
+
+In addition to the properties above, the `<Image />` component accepts all properties accepted by the HTML `<img>` tag.
+
+See more in the [Images Guide](/en/guides/images/#image--astroassets).
+
+### `<Picture />`
+
+<p><Since v="3.3.0" /></p>
+
+Use the built-in `<Picture />` Astro component to display a responsive image with multiple formats and/or sizes.
+
+```astro title="src/pages/index.astro"
+---
+import { Picture } from 'astro:assets';
+import myImage from "../assets/my_image.png"; // Image is 1600x900
+---
+
+<!-- `alt` is mandatory on the Picture component -->
+<Picture src={myImage} formats={['avif', 'webp']} alt="A description of my image." />
+```
+
+```html
+<!-- Output -->
+<picture>
+  <source srcset="/_astro/my_image.hash.avif" type="image/avif" />
+  <source srcset="/_astro/my_image.hash.webp" type="image/webp" />
+  <img
+    src="/_astro/my_image.hash.png"
+    width="1600"
+    height="900"
+    decoding="async"
+    loading="lazy"
+    alt="A description of my image."
+  />
+</picture>
+```
+
+See more in the [Images Guide](/en/guides/images/#picture-).
+
+#### Properties
+
+`<Picture />` accepts all the properties of the `<Image />` component, plus the following:
+
+##### `formats`
+
+An array of image formats to use for the `<source>` tags. By default, this is set to `['webp']`.
+
+##### `fallbackFormat`
+
+Format to use as a fallback value for the `<img>` tag. Defaults to `.png` for static images, `.gif` for animated images, and `.svg` for SVG files.
+
+##### `pictureAttributes`
+
+An object of attributes to be added to the `<picture>` tag. Use this property to apply attributes to the outer `<picture>` element itself. Attributes applied to the `<Picture />` component directly will apply to the inner `<img>` element, except for those used for image transformation.
+
+
+### `<Content />`
+
+A generic component used to render the content of a [content collection entry](/en/guides/content-collections/#what-are-content-collections).
+
+First, query one or more entries using `getCollection()` or `getEntry()`. Then, the `entry.render()` function can return the `<Content />` component for use in a `.astro` file template.
+
+```astro title="src/pages/render-example.astro" {4, 7}
+---
+import { getEntry } from 'astro:content';
+const entry = await getEntry('blog', 'post-1');
+const { Content } = await entry.render();
+---
+<p>Published on: {entry.data.published.toDateString()}</p>
+<Content />
+```
+
+### `<ViewTransitions />`
+
+<p><Since v="2.9.0" /></p>
+
+Opt in to using view transitions on individual pages by importing and adding the `<ViewTransitions />` routing component to `<head>` on every desired page.
+
+```astro title="src/pages/index.astro" ins={2,7}
```

Comment:
nit: All the code examples on the page use highlighting rather than diff notation so I think this one should do the same.
````suggestion
```astro title="src/pages/index.astro" {2,7}
````
Was there a specific reason for using `ins`? I'm guessing it was a copy/paste from an experimental feature/changeset.

Reply:
I'm going to guess that the `ins` was used to emphasize the "add this to otherwise unchanged code, and you get view transitions." Probably an artefact of this having been written when the component was new, and so all the text was about "adding" view transitions. (I do remember at the time we were HEAVY on this language because we were emphasizing you didn't just "turn them on", you added them to every page you wanted them on, or in a broader component, like a layout.

We can consider now that this is just "reference" and not "omg, here's how you add this new thing!" 😄

Reply:
Afterwards, it's not necessarily annoying... it was more a question of consistency. But this can also be explained by the sentence above (“Opt in”) which perhaps "pushes more to action" than other descriptions so it can stay like that! (as I said, I was nitpicking 😄 ).